frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -158,9 +156,7 @@ class DocumentEdit extends Component {
       fields: fields.other
     }]
     const activeSection = this.props.section || sections[0].slug
-    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => {
-      return document.validationErrors[field]
-    }).length
+    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => document.validationErrors[field]).length
```

Comment:
```js
const hasValidationErrors = Object.keys(document.validationErrors)
  .filter(field => document.validationErrors[field])
  .length
```

As per [chaining rules](https://github.com/dadi/publish/blob/master/docs/coding-guidelines.md#function-chaining-formats) in coding guidelines.